systemtest/src/test/java/io/strimzi/systemtest/cruisecontrol/CruiseControlConfigurationST.java

```diff
@@ -91,11 +94,15 @@ void testDeployAndUnDeployCruiseControl() throws IOException {
         LOGGER.info("Verifying that there is no configuration to CruiseControl metric reporter in Kafka ConfigMap");
         assertThrows(WaitException.class, () -> CruiseControlUtils.verifyCruiseControlMetricReporterConfigurationInKafkaConfigMapIsPresent(CruiseControlUtils.getKafkaCruiseControlMetricsReporterConfiguration(testStorage.getNamespaceName(), testStorage.getClusterName())));
 
-        // TODO https://github.com/strimzi/strimzi-kafka-operator/issues/8864
-        /*if (!Environment.isKRaftModeEnabled()) {
-            LOGGER.info("Cruise Control Topics will not be deleted and will stay in the Kafka cluster");
-            CruiseControlUtils.verifyThatCruiseControlTopicsArePresent(testStorage.getNamespaceName(), defaultBrokerReplicaCount);
-        }*/
+        final AdminClient adminClient = AdminClientTemplates.deployPlainAdminClient(
```

Comment:
This is what I don't want. Everything related to resources should go directly through ResourceMananger. And the get it via I think as this is deployment you would need to use `DeploymentResource.get...`similarly as we do for KafkaResource.